spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/wavefront/WavefrontProperties.java

```diff
@@ -261,15 +264,28 @@ public void setBatchSize(Integer batchSize) {
 	public static class Tracing {
 
 		/**
-		 * Application name. Defaults to 'spring.application.name'.
+		 * Application name used in {@link ApplicationTags}. Defaults to
```

Comment:
Just like in the previous case, I would call out that this is the Wavefront application, service, etc. name.